docker/docker-compose.env

```
@@ -36,4 +36,14 @@ export DOGECOIN_ADDRESS=mwjUmhAW68zCtgZpW5b1xD5g7MZew6xPV4
 # for which the private key is known by a test suite. This allows the test suite
 # access to plenty of testing funds.
 export ZCASH_PK=cNSVbbsAcBQ6BAmMr6yH6DLWr7QTDptHwdzpy4GYxGDkNZeKnczK
-export ZCASH_ADDRESS=tmCTReBSJEDMWfFCkXXPMSB3EfuPg6SE9dw
\ No newline at end of file
+export ZCASH_ADDRESS=tmCTReBSJEDMWfFCkXXPMSB3EfuPg6SE9dw
+
+#
+# DigiByte
+#
+
+# DigiByte Address that will receive mining rewards. Generally, this is set to an
+# address for which the private key is known by a test suite. This allows the test
+# suite access to plenty of testing funds.
+export DIGIBYTE_PK=edxsTg33sNnsXBPzBYB13aHRysups1SYoRfBkrYksDj9vkmNfdXo
+export DIGIBYTE_ADDRESS=dgbrt1qlpazxc2ghkaw35vzqz7ljf7ksjd5fm96wzckty
```

Looks like these are using mainnet encodings. Addresses need to be for regnet, so I suggest changing these exports to: 
```suggestion
export DIGIBYTE_PK=eaeBaSNZmieYKXBwbwjUwzRazT45kVwi9Rubb13RJ1NgfDEUXXH8
export DIGIBYTE_ADDRESS=snhzjRDRwL5LD4ognxvjaP2St8G3MEExQp
```